app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -699,7 +698,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
   };
 
   $scope.updateAuthStatus = function(updatedValue) {
-    $scope.angularForm[$scope.authType + '_auth_status'].$setViewValue(updatedValue);
+    $scope.angularForm[$scope.currentTab + '_auth_status'].$setViewValue(updatedValue);
```

Comment:
@AparnaKarve that change causes the 'virtualization' tab of the Container provider to fail.
`$scope.authType` returns `kubevirt`, which `$scope.currentTab` returns 'virtualization`.

There is `kubevirt_auth_status` but there isn't `virtualization_auth_status`, therefore attempt to validate the virtualization tab fails.

Reply:
@masayag Is this just a matter of renaming the `virtualization` tab to `kubevirt`?
Is it OK if we rename the tab to `kubevirt`? Because `$scope.currentTab` needs to match with the tab name that is specified in the haml.

Reply:
@AparnaKarve ack from my side for renaming the tab.

Reply:
The PR works fine for editing the Container Provider and for enabling the 'Virtualization' tab
However, we do allow user to view & change token for the Kubevirt provider under Infra --> Providers --> Edit
and that fails with the shown error in the screenshot:

![selection_869](https://user-images.githubusercontent.com/316242/47435705-d48b1980-d7ad-11e8-9e03-05172bfd1cf0.png)

the value of: ManageIQ.angular.scope.angularForm['default_auth_status'] is:
undefined
in this context, this one works: ManageIQ.angular.scope.angularForm['kubevirt_auth_status'].$setViewValue(true)
which isn't the conventional way it works for other providers - therefore we can just disable that option on this dialog and not allow changing the token(password) in this context, only under the virtualization tab.

So this PR fixes the blocked functionality of adding kubevirt provider and one gap to close.
I think this PR should be merged and the other issue should be addressed separately.